components/PageHeader.vue

```diff
@@ -47,7 +47,7 @@
       class="p-lg-0 align-items-start justify-content-lg-end flex-column flex-lg-row d-none d-lg-block"
       role="navigation"
     >
-      <PageNavigation />
+      <PageNavigation :links="$store.state['link-group'].data.mainNavigation.links" />
```

Comment:
Let's give the PageHeader a couple of computed properties using mapState, for mainNavigation and mobileNavigation so that these attributes can be reduced to e.g. `<PageNavigation :links="mainNavigation.links" />`